lib/commands/general.js

```diff
@@ -0,0 +1,18 @@
+let commands = {};
+
+/**
+ * Close app (simulate device home button). If duration, app will re-open
+ * after that duration
+ */
+commands.background = async function (duration) {
```

Comment:
yup. All the functions that get exported from the "commands" directory get added to the XCUITestDriver object. so `this` refers to the main object.